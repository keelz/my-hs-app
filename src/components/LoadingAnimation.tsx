import * as React from 'react';

const LoadingAnimation: React.SFC = () =>
  <div className="loading-animation-container">
    <div className="loading-animation" />
  </div>;

export default LoadingAnimation;
