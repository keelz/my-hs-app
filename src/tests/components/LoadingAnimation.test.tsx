import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import LoadingAnimation from '../../components/LoadingAnimation';

describe('LoadingAnimation', () => {
  describe('snapshots', () => {
    it('renders correctly', () => {
      const tree = enzyme.shallow(<LoadingAnimation />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
