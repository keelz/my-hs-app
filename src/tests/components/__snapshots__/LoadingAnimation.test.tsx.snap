// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingAnimation snapshots renders correctly 1`] = `
<div
  className="loading-animation-container"
>
  <div
    className="loading-animation"
  />
</div>
`;
